packages/react-components/src/button/src/ButtonGroup.tsx

```diff
@@ -43,9 +46,9 @@ const Gap = {
         "sm": 2,
         "md": 3
     }
-};
+} as const;
```

Comment:
I guess we are using const here instead of a nesting of Records because it's easier and no one cares if we go with something more precise?

Reply:
Exactly, since its a local const, i could do a Record<"horizontal" | "vertical", { sm: number, md: number}> i prefer the const synthax in this case